pom.xml

```diff
@@ -2081,7 +2081,8 @@
 
         <bouncycastle.version>1.68</bouncycastle.version>
         <commons-lang3.version>3.3.2</commons-lang3.version>
-        <microprofile.config.version>2.0</microprofile.config.version>
+        <microprofile.config.version>3.0.1</microprofile.config.version>
```

Comment:
We probably should not update the config version to 3.x, there is no implementation for it out there, yet.